blocks/collections-content-source-block/package.json

```diff
@@ -26,5 +26,8 @@
     "@wpmedia/engine-theme-sdk": "*",
     "@wpmedia/resizer-image-block": "*"
   },
-  "gitHead": "83fb8d6685958d85ddbfcfbf01d9a5c864cd6c95"
+  "gitHead": "83fb8d6685958d85ddbfcfbf01d9a5c864cd6c95",
+  "dependencies": {
+    "request-promise-native": "^1.0.9"
```

Comment:
Not a blocker, but more so a general question - Have we considered using another HTTP client that handles promises? `request-promise-native` is deprecated and libraries like `axios` are just as capable and smaller.
https://www.npmtrends.com/axios-vs-request-promise-native

Reply:
@ecarlisle I've switched to use `axios` now, this was my bad - I use the library that was referenced in the ALC documentation for content sources